projects/uitpas/docs/errors.md

```diff
@@ -0,0 +1,341 @@
+
+## invalid-card
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/invalid-card`
+* **Title**: `Invalid card`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the given card is not applicable in this context.
+
+## invalid-card-status
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/invalid-card-status`
+* **Title**: `Invalid card status`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the given card is not in the expected state.
+
+## maximum-reached
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/maximum-reached`
+* **Title**: `Maximum reached`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the maximum number of operations of this kind have been reached.
+
+## invalid-coupon
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/invalid-coupon`
+* **Title**: `Invalid coupon`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the given coupon is not valid.
+
+## social-tariff-expired
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/social-tariff-expired`
+* **Title**: `Social tariff expired`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the passholder has an expired social tariff.
+
+## passholder-no-active-cardsystems
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/passholder-no-active-cardsystems`
+* **Title**: `Passholder has no active cardsystem memberships`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the passholder has no active card system memberships.
+
+## ticket-sale-not-found
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/ticket-sale-not-found`
+* **Title**: `Ticket sale not found`
+* **Status**: `404`
+
+The request you are trying to perform cannot complete because the given ticket sale cannot be found.
+
+## invalid-city
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/invalid-city`
+* **Title**: `Invalid city`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the provided city is invalid.
+
+## invalid-postal-code
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/invalid-postal-code`
+* **Title**: `Invalid postal code`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the provided postal code is invalid.
+
+## insz-in-token-required
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/insz-in-token-required`
+* **Title**: `INSZ in token required`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because this request needs a token including an `https://publiq.be/rrn` claim.
+
+## forbidden-underaged
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/forbidden-underaged`
+* **Title**: `Forbidden Underaged`
+* **Status**: `400`
```

Comment:
`403` might be better for Forbidden, or even `451 (Unavailable For Legal Reasons)`?

Reply:
I don't like 403 for this case.
I'm not against 451 but not sure if that is the intended use

Reply:
Also okay to leave it like this for me, it was just a minor suggestion